src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/CSharpFormatter.cs

```diff
@@ -99,46 +83,35 @@ internal class CSharpFormatter
             return mappedEdits;
         }
 
-        public int GetCSharpIndentation(FormattingContext context, int projectedDocumentIndex, CancellationToken cancellationToken)
+        public async Task<int> GetCSharpIndentationAsync(FormattingContext context, int projectedDocumentIndex, CancellationToken cancellationToken)
```

Comment:
This method is currently ugly because it formats and ToString()s everytime. The way to avoid it would be to use syntax tree annotations but there seems to be a bug that is preventing us from using it. See https://github.com/dotnet/roslyn/issues/49492.